docs/document/website/versioned_docs/version-1.12.0/components/desktop/tooltip.md

```diff
@@ -0,0 +1,94 @@
+
+> *1: Please use **focusable elements** as much as possible and avoid abusing unfocusable elements for accessibility.
+
+> *2: By understanding how to use the describeChild property, you can effectively improve the accessibility and usability of your application.
+> - Set `describeChild` to `false` if the Tooltip represents the purpose of the element itself (provides a brief explanation or label for the element it is attached to).
+>   - The `aria-label` attribute will be added to the first child element of the Tooltip component with the value of the `title` property.
+>   - For example, if you have a button with a Tooltip that says "Save", the Tooltip represents the purpose of the button itself. In this case, set `describeChild` to `false` to indicate that the Tooltip explains the purpose of the button.
+>     ```javascript
+>       <kuc-tooltip>
+>         <button aria-label=“Save”>Save</button>
+>         <div id="tooltip-ID">Save</div>
+>       </kuc-tooltip>
+>     ```
+>    - You can learn more about [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label).
+> - Set `describeChild` to `true` if the Tooltip represents the description of the element (provides additional information or a description about the element it is attached to).
+>   - The `aria-describedby` attribute will be added to the first child element of the Tooltip component with the value of the ID of the tooltip wrapper element.
+>   - For example, if you have an icon that represents a question mark, and the Tooltip provides a description or explanation of what the icon represents. In this case, set `describeChild` to `true` to indicate that the Tooltip serves as a description of the icon.
+>     ```javascript
+>       <kuc-tooltip>
+>         <button aria-describedby=“tooltip-ID”>
+>           <span class="icon-question-mark"></span>
+>         </button>
+>         <div id="tooltip-ID">This is a help icon. Click for more information</div>
+>       </kuc-tooltip>
+>     ```
+>   - You can learn more about [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby).
+
+### Constructor
+
+Tooltip(options)<br>
+Here is a list of available constructors:
+
+#### Parameter
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| options  | object | {} | Object that includes component properties |  |
+
+---
+
+## Sample Code
+
+> Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
+
+Here is a sample code when all parameters are specified:
+
+```javascript
+const Kuc = Kucs['1.x.x'];
+
+const space = kintone.app.record.getSpaceElement('space');
+const buttonEl = document.createElement('button');
+buttonEl.innerText ='Add';
+
+const tooltip = new Kuc.Tooltip({
+  title: "Do not add if it exists.",
+  container: buttonEl,
+  placement: 'bottom',
+  describeChild: false,
```

Comment:
Is it ok with false in this case?
I think the tooltip role this time will be the explanation, so it  should be true.

Reply:
Yes, I will update it to true